src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -237,6 +237,18 @@ public String getAwsSecretKey() {
     public String getAwsEndpoint() {
         return getString("aws.endpoint");
     }
+    
+    public boolean getAwsProxyEnabled(){
+    	return getBoolean("aws.proxy.isEnabled", false);
```

Comment:
for the boolean property, the method getBoolean expects 2 parameter, properyName and the defaultValue.